Jenkinsfile

```diff
@@ -87,12 +87,65 @@ pipeline {
         Execute unit tests.
         */
         stage('Test') {
-          steps {
-            withGithubNotify(context: 'Test', tab: 'tests') {
-              deleteDir()
-              unstash 'source'
-              dir("${BASE_DIR}"){
-                runParallelTest()
+          matrix {
+            agent { label 'linux && immutable' }
+            axes {
+                axis {
+                    name 'STACK_VERSION'
+                    values (
+                      '8.0.0-SNAPSHOT',
+                      '7.0.0',
```

Comment:
can we keep the cross browser tests only for a specific stack? it increases the matrix by 3x. Instead we can run the sauce labs tests only for a stack like how we do in travis and only run chrome headless tests in other stacks. It helps reduce the test time as well as sauce timeouts. WDYT @kuisathaverat

Reply:
yep, make sense, which command I have to use?

Reply:
toggling the mode should do the trick, MODE=saucelabs for all browsers and MODE=none should run only on chrome.

Reply:
I don't think we should do this though, we have seen errors happing on older stack different browser version because the cross origin issue that needed to be fixed in the APM server.

I don't think the number of tests is the issue here, the issue is with concurrency regardless of the number of jobs.

Reply:
if we can make it faster, then I am all in for running on different stacks. However we would have to remove running in 6.5 anyways since we wont be supporting it moving forward. So just changing it to >7.0 would reduce the time ass well.